package.json

```diff
@@ -2,7 +2,7 @@
     "name": "mindaro",
     "displayName": "Bridge to Kubernetes",
     "description": "Rapid Kubernetes development for teams",
-    "version": "2.0.120230716",
+    "version": "2.0.120230817",
```

Comment:
💡☕️ this is only needed in case of release, we can avoid upgrading the version number form this.

Just for thought/idea share - we could also think internally if this is major or minor release which will impact the version number. 🙏☕️

Reply:
Yeah, I updated the version number to keep track of the logs after replacing the telemetry. I have given the vsix file to the testing team, if everything looks good, I will revert the version number before merging.

Reply:
It is working fine in all the environments, so I have reverted back to the old version.